spec/factories/freak.rb

```diff
@@ -1,10 +1,16 @@
 # frozen_string_literal: true
 
+# frozen_string_literal: truex``
```

Comment:
I suggest doing a self code review after pushing new changes, some things will stand out to you as obviously needing changes.

Reply:
Thanks for the advice, I'll be more careful :slightly_smiling_face: